stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/Message.kt

```diff
@@ -17,36 +17,36 @@ internal fun Message.getSenderDisplayName(context: Context, isDirectMessaging: B
 private fun getSelfDisplayName(context: Context) =
     context.getString(R.string.stream_ui_channel_display_name_self)
 
-internal fun Message.isDeleted(): Boolean = deletedAt != null
+public fun Message.isDeleted(): Boolean = deletedAt != null
```

Comment:
I'd leave these as internal API if possible, clients can always access the underlying data themselves if they really want to, and we'd have less API to worry about

Reply:
From my point of view, these extensions look simpler and clearer than raw data inside models. There are no hidden implementation details, I think is entirely ok to make them public

Reply:
Alright, can we please rename the last three of them though? 😄 

They have really awkward names for public API, could be `hasSingleReaction`, `isReply`, and `hasText` instead.

Reply:
No problem)